src/Authenticators/LDAPAuthenticator.php

```diff
@@ -148,7 +148,6 @@ public function authenticate(array $data, HTTPRequest $request, ValidationResult
             return null;
         }
 
-        // LDAPMemberExtension::memberLoggedIn() will update any other AD attributes mapped to Member fields
```

Comment:
Could you keep this comment and change it to `LDAPMemberExtension::afterMemberLoggedIn()` - that extension hook is still being used

Reply:
Ok. I was confused since it was marked with **_"Will be removed without equivalent functionality to replace it"_** message.

Reply:
DONE